guides/ssb-server-context.md

```diff
@@ -0,0 +1,36 @@
+# ssb-server, ssb-client & Core Functions
+
+## What is ssb-server?
+[ssb-server](https://github.com/ssbc/ssb-server) is the component of secure-scuttlebutt that performs the peer-to-peer connections, and reading and writing to/from the append-only log. It can also be extended with plugins. It exposes via its API core functionality such as reading from, and writing to, the append-only log, as well as the extended functions provided by plugins utilized. This process should be thought of as a sort of "background" process, since it can only perform the peer-to-peer "gossiping" functions if it is running. An ssb-server is one to one coupled with an "identity" which is essentially the secret key associated with a given append-only log.
+
+> Sometimes, ssb-server is packaged as an "embedded" process with an SSB application, such as Patchwork. Two or more versions of ssb-server connecting to the same identity and feed (log) should not run at the same time. The SSB ecosystem is currently exploring a variety of solutions to this problem.
```

Comment:
"at the same time" could be more clear.
perhaps "you can only safely run one application at a given time which interacts with your identity. You can stop one application and start another one, and that's fine. The SSB ecosystem..."